client/pages/matches.jsx

```diff
@@ -43,6 +45,29 @@ export default class Matches extends React.Component {
       });
   }
 
+  handleSave() {
+    fetch('/api/favorites', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        petId: Number(this.state.id),
+        userId: 1,
+        details: {
+          name: this.state.name,
+          photos: this.state.photos,
+          location: this.state.location,
+          age: this.state.age,
+          breed: this.state.breed
+        }
+      })
+    })
+      .then(response => response.json())
+      .catch(error => {
+        console.error('Error', error);
+      });
+    refreshPage();
```

Comment:
I don't think this refresh page is necessary. What state are you trying to reset? Do that instead.